src/main/io/vtx_smartaudio.c

```diff
@@ -86,7 +86,8 @@ enum {
     SA_CMD_SET_CHAN,
     SA_CMD_SET_FREQ,
     SA_CMD_SET_MODE,
-    SA_CMD_GET_SETTINGS_V2 = 0x09        // Response only
+    SA_CMD_GET_SETTINGS_V2 = 0x09,        // Response only
+    SA_CMD_GET_SETTINGS_V21 = 0x11
```

Comment:
Add a comma to the end of this, and next time the review will only show the changed line...